src/App.js

```diff
@@ -16,6 +17,9 @@ export default function App() {
             <StyledLi>
               <StyledLink to="/">Home</StyledLink>
             </StyledLi>
+            <StyledLi>
+              <SearchComponent />
```

Comment:
not sure if I want to have Component in my name, it's like having additionFunction, you know what I mean?

Reply:
Good point. Found a way to rename it.